app/services/proxy_load_balancer.rb

```diff
@@ -0,0 +1,28 @@
+class ProxyLoadBalancer
+  def initialize
+    all_proxies = SmartProxy.unscoped.with_features(RemoteExecutionProvider.providers.keys)
+    @jobs = get_counts(all_proxies)
```

Comment:
I mean calculating them the first time the `next` method is called with the given proxy